docs/installing.rst

```diff
@@ -3,40 +3,59 @@
 Installation
 ============
 
-Python & OS Support
--------------------
-
-pip works with CPython versions 2.6, 2.7, 3.2, 3.3, 3.4 and also pypy.
-
-pip works on Unix/Linux, OS X, and Windows.
+Do I need to install pip?
+-------------------------
 
-.. note::
+Distributions of Python 2.7.9 and later (in the Python 2 series), and
+Python 3.4 and later (in the Python 3 series) may already include pip by
+default. [1]_
 
-  Python 2.5 was supported through v1.3.1, and Python 2.4 was supported through v1.1.
+Additionally, it's common to be working in a :ref:`Virtual Envionment
+<pypug:Creating and using Virtual Environments>` created by a tool like
+:ref:`pypug:virtualenv` or :ref:`pyvenv <pypug:venv>`, which handles installing
+pip for you.
 
-pip included with Python
-------------------------
-Python 2.7.9 and later (on the python2 series), and Python 3.4
-and later include pip by default [1]_, so you may have pip already.
 
 .. _`get-pip`:
 
-Install pip
------------
+Installing with get-pip.py
+--------------------------
 
 To install pip, securely download `get-pip.py
 <https://bootstrap.pypa.io/get-pip.py>`_. [2]_
 
-Then run the following (which may require administrator access):
+Then run the following:
 
 ::
 
  python get-pip.py
 
-If `setuptools`_ is not already installed, ``get-pip.py`` will install
-`setuptools`_ for you. [3]_
 
-To upgrade an existing `setuptools`_, run ``pip install -U setuptools``.
+.. warning::
+
+   If you're using a Python install that's managed by your operating system or
+   another package manager, then be cautious. "get-pip.py" does not coordinate
+   with those tools, and may leave your system in an inconsistent state.
+
+get-pip.py will also intall :ref:`pypug:setuptools` [3]_ and :ref:`pypug:wheel`,
+if they're not already. :ref:`pypug:setuptools` is required to install
```

Comment:
Probably better to avoid the subordinate clause. "If they are not already installed pip will install :... and :....`.